services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -153,26 +213,37 @@ public static Builder builder() {
   public abstract Boolean roundaboutExits();
 
   /**
-   * Geometry type used to make the initial directions request.
+   * The format of the returned geometry. Allowed values are:
+   * {@link DirectionsCriteria#GEOMETRY_POLYLINE} (default, a polyline with a precision of five
+   * decimal places), {@link DirectionsCriteria#GEOMETRY_POLYLINE6} (a polyline with a precision
+   * of six decimal places).
+   * A null value will reset this field to the APIs default value vs this SDKs default value of
+   * {@link DirectionsCriteria#GEOMETRY_POLYLINE6}.
```

Comment:
Sentences like this should only be in the builder methods, not getters. Same with other instances that mention `take`/`specify`/`set` instead of `return` etc. This is not a big deal though.

Reply:
fixed